src/Opdex.Platform.Infrastructure.Abstractions/Clients/CirrusFullNodeApi/Commands/CallCirrusCallSmartContractMethodCommand.cs

```diff
@@ -1,26 +1,26 @@
 using MediatR;
 using Opdex.Platform.Common.Models;
-using Opdex.Platform.Domain.Models.Transactions;
 using Opdex.Platform.Infrastructure.Abstractions.Clients.CirrusFullNodeApi.Models;
 using System;
 
 namespace Opdex.Platform.Infrastructure.Abstractions.Clients.CirrusFullNodeApi.Commands
 {
+    /// <summary>
+    /// Generic command to call a smart contract and broadcast the transaction. Intended only for contract deployment.
+    /// </summary>
     public class CallCirrusCallSmartContractMethodCommand : IRequest<Sha256>
     {
-        public CallCirrusCallSmartContractMethodCommand(TransactionQuoteRequest quoteRequest = null, SmartContractCallRequestDto callDto = null)
```

Comment:
Nice to fix this up 👍 I was confused at how you did it at first, but I see now :)